src/lib/user-ip-address.js

```diff
@@ -0,0 +1,16 @@
+module.exports = (req) => {
+  const xHeaderIpAddress = req.headers["x-forwarded-for"];
+
+  let ipAddress;
+  if (xHeaderIpAddress !== undefined) {
+    ipAddress = xHeaderIpAddress.split(",")[0];
+  } else {
+    ipAddress =
+      (req.connection && req.connection.remoteAddress) ||
```

Comment:
I'd like a bit more certainty about what properties we're using, so let's do some pairing in a separate branch on investigating this, and then pull what we learned into this code here.